packages/firestore/LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright 2017, the Flutter project authors. All rights reserved.
```

Comment:
Unify copyright lines? Here and in the source file headers?

Some source file headers say "Chromium authors", which is what we use for the other plugins as well, I think.

Reply:
Fixed